.github/workflows/deploy-person-api.dev.yml

```diff
@@ -0,0 +1,48 @@
+name: Deploy person-api t2
+
+on:
+  push:
```

Comment:
Legge til master igjen siden dette er for t miljø.
on:
  push:
    branches: master
    paths:
      - apps/person-api/**
      - libs/testnorge-core/**